public/manifest.json

```
@@ -2,7 +2,7 @@
   "short_name": "API Docs",
   "name": "API Docs",
   "icons": [],
-  "start_url": "/apidocs",
+  "start_url": ".",
```

as per the docs, all from `public` can use variables 

```suggestion
  "start_url": "%PUBLIC_URL%",
```